web/src/features/moreCast2/components/TabbedDataGrid.tsx

```diff
@@ -350,22 +344,42 @@ const TabbedDataGrid = ({ morecast2Rows, fromTo, setFromTo, fetchWeatherIndeterm
 
   /********** End useEffects for managing visibility of column groups *************/
 
+  const updateColumnHelper = (editedRows: MoreCast2Row[]) => {
+    storedDraftForecast.updateStoredDraftForecasts(editedRows, getDateTimeNowPST())
```

Comment:
Thinking this should maybe happen at the end of this function so the stored draft has the correct simulated indices? If I go in and edit a forecast, then refresh the page, morecast loads with the incorrect indices

Reply:
Good catch, thank you!